doc-content/jbpm-docs/src/main/asciidoc/BPMN2/process-instance-admin-con.adoc

```diff
@@ -1,5 +1,8 @@
 [id='process-instance-admin-con']
-= Process instance administration
+= Process instance operations
+
+*<MH>Dev - I am clicking around and do not see where these "operations" are located within a process instance. Is this chapter useful?</MH>*
```

Comment:
This is actually not about the UI, but the process instance admin api that is exposed on kie-server:
http://mswiderski.blogspot.com/2016/11/administration-interfaces-in-jbpm-7.html

Reply:
OK Kris. That makes sense. I am extremely busy wrapping up other docs for 7.8, so I will simply remove my note for now. I would like to revisit this doc post GA to ensure a better experience. I find it very clunky, but no time for polishing it now. Thank you for the details.